src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -387,8 +508,8 @@ BtreeLBAManager::scan_mappings(
 	      seastar::stop_iteration::yes);
 	  }
 	  ceph_assert((pos.get_key() + pos.get_val().len) > begin);
```

Comment:
Add `ceph_assert(pos.get_val().pladdr.is_paddr())` before assuming it is an paddr?

But looks `BtreeLBAManager::scan_mappings()` is no longer used ?

Reply:
`pladdr_t::get_paddr()` has this assertion